source/elements/oneMKL/source/domains/spblas/operations/spmv.rst

```diff
@@ -292,7 +294,11 @@ spmv
 
    A_view
       Specifies which part of the handle should be read as described by
-      :ref:`onemkl_sparse_matrix_view`.
+      :ref:`onemkl_sparse_matrix_view`. The ``type_view`` field cannot be
+      ``matrix_descr::diagonal``. The ``diag_view`` field can be ``diag::unit``
+      if and only if ``type_view`` is ``matrix_descr::triangular``. The
+      ``type_view`` field cannot be ``matrix_descr::symmetric`` nor
+      ``matrix_descr::hermitian`` if ``opA`` is ``transpose::conjtrans``.
```

Comment:
```suggestion
      ``matrix_descr::hermitian`` if ``opA`` is ``transpose::conjtrans``.
      Support may be expanded in the future.
```

Reply:
I have answered this in the comment above https://github.com/uxlfoundation/oneAPI-spec/pull/542#discussion_r1723195090